modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/CheckboxMultiple/CheckboxMultiple.es.js

```diff
@@ -128,7 +129,8 @@ const Main = ({
 	value,
 	...otherProps
 }) => (
-	<FieldBase name={name} readOnly={readOnly} {...otherProps}>
+	<FieldBase {...otherProps} name={name} readOnly={readOnly} showLabel={false}>
+		<legend className="lfr-ddm-legend">{label}</legend>
```

Comment:
@antonio-ortega, can you please clarify to me the purpose of add a `<legend>` here once we have no `<fieldset>`. More details in https://developer.mozilla.org/en-US/docs/Web/HTML/Element/legend.

Reply:
@antonio-ortega, any news about the comment above?

Reply:
@natocesarrego Sorry for the delay. I missed this comment. The purpose of adding the legend was to be closer to what is expected from an accessibility point of view (see [here](https://www.w3.org/WAI/tutorials/forms/grouping/#radio-buttons) and [here](https://webaim.org/techniques/forms/controls#radio), and even in [this internal PTR](https://issues.liferay.com/browse/PTR-1845?focusedCommentId=2206043&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-2206043), Patrick also mentioned the possibility of adding a fieldset&legend).

My bad because I added the legend field but not the fieldset, I can work to do it in case this suits the way we want to fix this issue.

Reply:
No problem, @antonio-ortega! 😄

Please add the `fieldset` tag. Recently, a TSE from Brazil did the same to address another Accessibility issue. 🙂